CMakeLists.txt

```diff
@@ -344,6 +344,9 @@ if (WIN32)
   set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} /DEBUG /PDBCOMPRESS")
   set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} /STACK:1572864")
 
+  # Disable incremental link due to incremental linking CFG bug crashing crossgen
```

Comment:
># Disable incremental link due to incremental linking CFG bug crashing crossgen [](start = 2, length = 79)

Can you file a tracking bug, so that when the linker gets fixed and updated in LKG, we can restore this back?

Also, if you can add a small comment with a brief description of what the bug was, and the bug ID tracking the undoing of this change, that would be awesome :)